### PR TITLE
Notifications demo

### PR DESCRIPTION
**Pull request by caupcakes**
- store token to send if offline
- reconfig app.json for projectid

### CODE REVIEW

**Comment by AydanPirani** [comment]
hihi, got it! couple questions:

1) how does expo handle sending notifications to multiple devices?
2) is it possible for all this to happen async behind the scenes?

**Comment by caupcakes** [comment]
1.I believe each token may be device specific - not sure about the details but it should not matter unless someone is signed in with the same apple id / google login on different devices.
2. Yeah - if you have the token you can write purely backend code to send whatever you want. A setup could be storing only Expo tokens, and then having some script send a push to all the tokens if R|P is 24 hrs away for example.

**Comment by AydanPirani** [comment]
sure thing, lgtm (for the most part)! @caupcakes  can you please fix merge conflicts + remove the .idea files (and add the .idea folder to the gitignore)?

will approve right after

File: app.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "expo": {
     "name": "rp-mobile-2024",
-    "slug": "rp-mobile-2024",
+    "slug": "rorp-demo",
```
Comment (AydanPirani) [inline app.json:4]: unchange pls

File: app.json [hunk 2]
```diff
@@ -25,6 +25,12 @@
...
     "web": {
       "favicon": "./assets/favicon.png"
-    }
+    },
+    "extra": {
+      "eas": {
+        "projectId": "0ea8d500-a6ae-43f1-a262-51a05ccceed9"
+      }
+    },
+    "owner": "jeremywubv"
```
Comment (AydanPirani) [inline app.json:34]: is this because we have an expo portal login somewhere? if so, lmk and i'll make an RP one

Comment (caupcakes) [reply app.json:34]: I dont know the r|p expo login so i made a demo project to test notifications

File: app.json (hunk 1 above)
Comment (caupcakes) [reply app.json]: done

**Review by AydanPirani — DISMISSED** [review]

**Comment by AydanPirani** [comment]
lgtm! fix merge conflicts and merge pls :)

**Review by AydanPirani — APPROVED** [review]

**Review by AydanPirani — APPROVED** [review]

**Review by AydanPirani — APPROVED** [review]

**Pull request merged**